### PR TITLE
Tickets > Move action to proper place.

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://moderntribe.atlassian.net/browse/ETP-302

Move action after the ticket was created, so it works also on ticket creation.

### CODE REVIEW

File: src/Tribe/Tickets.php [hunk 1]
```diff
@@ -2842,6 +2842,9 @@ public function ticket_add( $post_id, $data ) {
 				$ticket->end_date = date( Tribe__Date_Utils::DBDATEFORMAT, strtotime( $end_datetime ) );
 			}
 
+			// Pass the control to the child object
```
Comment (tr1b0t) [inline src/Tribe/Tickets.php]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 1e0a8762b1c11a6aae6293cca2eb0fe9</sub>

**Comment by bordoni** [comment]
That is a good fix! Would you mind laying out steps that QA could test this? Would make it easier to make sure we can get it into the next release.

**Comment by juanfra** [comment]
Well, there's nothing really to test from a QA perspective, it's just moving the action so the ticket is created when it's being triggered.

Maybe adding this to their functions.php file and checking that the ID is in place with the fix:

`add_action( 'tribe_tickets_ticket_add', 'check_ticket_data_on_ticket_add', 10, 3 );

function check_ticket_data_on_ticket_add( $post_id, $ticket, $data ) {

	error_log( var_export( $ticket->ID, true ) );

}`

**Comment by sc0ttkclark** [comment]
Please update ticket with testing instructions for QA to use

**Pull request merged**